client/src/components/App.jsx

```diff
@@ -1,12 +1,35 @@
 
 import React from 'react';
+import '../main.css';
+import images from './dummydata';
+import Carousel from './Carousel';
 
-const App = () => (
-  <div>
-    <nav>
-      <h1>Nerdstrom</h1>
-    </nav>
-  </div>
-);
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      products: images,
+      mainImage: images[0].main,
+    };
+  }
+
+  changeMainImage(img) {
+    this.setState = {
```

Comment:
this.setState is a function, should look like this: 
``this.setState({
  mainImage: img,
})``